src/ui/server/test-mocks/mock-application.ts

```diff
@@ -5,6 +5,7 @@ import mockCountries from './mock-countries';
 import mockCurrencies from './mock-currencies';
 import { Application } from '../../types';
 import mockBroker from './mock-broker';
+import mockBuyerExample from './mock-buyer';
```

Comment:
`mockBuyer`

Reply:
used mockBuyerExample as mockBuyer is being used below

Reply:
Maybe we can do this - rename `mockBuyerExample` to `mockBuyer`, and rename line 81 to `mockApplicationBuyer`?